core/src/test/java/org/frankframework/senders/JavascriptSenderCallbackTest.java

```diff
@@ -24,6 +28,11 @@
 
 class JavascriptSenderCallbackTest extends SenderTestBase<JavascriptSender> {
 
+	@BeforeAll
+	public static void setup() {
+		assumeFalse(TestAssertions.isTestRunningOnARM());
```

Comment:
```suggestion
		// Since it's not possible to conditionally check the EnumSource here, just disable all for now...
		assumeFalse(TestAssertions.isTestRunningOnARM(), "uses J2V8 which does not work on ARM");
```